### ISSUE TITLE
Defining custom enviroment variables

### ISSUE THREAD

**Issue by wouter2397** [pegasystems/pega-helm-charts#39]
Hello,

I've been using this Pega helm chart for some weeks now.
I'm currently working on tweeking the config for our environment.

Now I need to change the timezone on every container to Europe/Amsterdam.
I've tested this by adding an enviroment variable 'TZ' and this worked perfectly.

Next I want to change the helm configuration so this enviroment will be added in every deploymentconfig.

I found the following code in file templates/_pega-deployment.tpl

`{{- if .custom }}
{{- if .custom.env }}
        # Additional custom env vars
{{ toYaml .custom.env | indent 8 }}
{{- end }}`

The problem is that I can't find anything regarding setting .custom.env. Could you describe to me how I can configure these custom variables inside this helm chart?

**Comment by Emilio-Pega** [comment]
Hmm this info *used* to be documented in the values.yaml, but looks like it was removed

But basically for a tier:
```
tier:
  - name: my-tier
    custom:
      env:
         #standard k8s yaml for an env var
         - name: MY_ENV_NAME
           value: MY_ENV_VALUE
```

**Comment by wouter2397** [comment]
Thank you for your quick reply. Today I had no time to change my configuration.
I hope to test this tomorrow.

For now thanks for your support!

**Comment by dcasavant** [comment]
It does look like this use case was lost at some point.  I created a branch with the documentation change staged.  Will wait for verification from @wouter2397.  If any chart changes are required to functionally support this use case, we can add those as well.

Another temporary workaround would also be to use a [prconfig override](https://github.com/pegasystems/pega-helm-charts/tree/master/charts/pega#pega-configuration-files) to pass the variable in.

**Comment by wouter2397** [comment]
So I tested the configuration from @Emilio-Pega. 
Everything is working fine for the Pega tier nodes.

Only the search and cassandra nodes are not changed yet. (ofcouse)
What would be the best way to add custom variables for those nodes?
Most ideal would be to add those to the same values.yaml.

**Comment by Emilio-Pega** [comment]
For cassandra, that is a chart we just pull in from google, so you would want to look at that [chart's documentation](https://github.com/helm/charts/tree/master/incubator/cassandra).  You can use any of the configuration described there.

Not sure about search...

**Comment by dcasavant** [comment]
The pegasearch yaml [is defined here](https://github.com/pegasystems/pega-helm-charts/blob/master/charts/pega/charts/pegasearch/values.yaml) but does not allow for freeform env variables.  What is the use case?

**Comment by wouter2397** [comment]
Usecase will be the same as for my original issue. Adding an enviroment variable 'TZ' so I can change the timezone of the pegasearch and cassandra containers.

I thought maybe it would be possible to do this within the 'master' values file, but apparently it's not.
I just took a look at the documentation of both other charts and found a possible for my use case in both.

So next Monday I will investigate this further..
For now thank you both and I will keep you updated!

**Comment by wouter2397** [comment]
So another update from my side:
I was able to set the timezone for both pegasearch and cassandra charts.

For the pegasearch I had to edit the file: charts/pegasearch/templates/pega-search-deployment.yaml.
I added the following configuration:
```yaml
        env:
        - name: TZ
          value: "{{ .Values.env.Timezone }}"
```

With this setup I have to edit the template files everytime I want to add an enviroment variable. Is it possible to apply the same structure as the general pega chart with the 'custom' section?
